src/components/Card.js

```diff
@@ -19,9 +11,8 @@ export default class Card extends Component {
 
   constructor(properties) {
     super(properties);
-    this.state = {
-      childProps: sanitizeChildProps(properties)
-    };
+    const { style, ...childProps } = properties;
+    this.childProps = childProps;
```

Comment:
I wasn't sure about this myself for quite some time, but when the component receives new props render will be called anyway. see https://facebook.github.io/react/docs/component-specs.html#updating-componentwillreceiveprops